netbox_agent/location.py

```diff
@@ -0,0 +1,71 @@
+import importlib
+import importlib.machinery
+
+from netbox_agent.config import DATACENTER_LOCATION, DATACENTER_LOCATION_DRIVER_FILE, \
+    DATACENTER_LOCATION_REGEX, RACK_LOCATION, RACK_LOCATION_DRIVER_FILE, RACK_LOCATION_REGEX
+
+
+class LocationBase():
+    """
+    This class is used to guess the location in order to push the information
+    in Netbox for a `Device`
+
+    A driver takes a `value` and evaluates a regex with a `capture group`.
+
+    There's embeded drivers such as `file` or `cmd` which read a file or return the
+    output of a file.
+
+    There's also a support for an external driver file outside of this project in case
+    the logic isn't supported here.
```

Comment:
c'est quoi le use case à partir du moment où tu peux déjà faire `cmd: python foo.py` ?

Reply:
good point, I'll leave it that way for the moment
I find it cleaner to execute the python-way instead of fork